### PR TITLE
Add Slack notification to help-p1 when action fails

### PR DESCRIPTION
**Pull request by lucasmrod**
Copied from [fleetdm/nvd](https://github.com/fleetdm/nvd):
https://github.com/fleetdm/nvd/blob/dc46bfca2435e11df65b1ec9e7ef579e71904a0a/.github/workflows/generate.yml#L76-L95

@lukeheath We'd need `secrets.SLACK_G_HELP_P1_WEBHOOK_URL` also defined on this repository.

### CODE REVIEW

**Review by mostlikelee — APPROVED** [review]
I think this is fine for the first iteration, but we should decide if alerting on a single failure is going to be too noisy.  I believe we may be discussing SLA/SLOs on Thursday.

**Comment by lucasmrod** [comment]
Agree, we can revert if it becomes too noisy.

I'll wait for Luke to create the secret before merging.

**Comment by lukeheath** [comment]
@lucasmrod That is an organization secret, so it is available in all repos.

**Pull request merged**
